app/components/security/analysis-list/table/action/index.ts

```diff
@@ -109,15 +78,15 @@ export default class AnalysisOverviewComponent extends Component {
           nistsp80053: this.analysis.nistsp80053.map((a) => a.get('id')),
           nistsp800171: this.analysis.nistsp800171.map((a) => a.get('id')),
           findings: this.analysis.findings,
-          overridden_risk: this.analysis.overridden_risk || 'None',
-          overridden_risk_comment: this.analysis.overridden_risk_comment || '',
+          overridden_risk: this.analysis.overriddenRisk || 'None',
+          overridden_risk_comment: this.analysis.overriddenRiskComment || '',
           overridden_risk_to_profile:
-            this.analysis.overridden_risk_to_profile || false,
+            this.analysis.overriddenRiskToProfile || false,
         }),
       });
 
       this.analysis.risk = this.PASSED_STATE.risk;
-      this.analysis.status = this.PASSED_STATE.status;
+      this.analysis.status = String(this.PASSED_STATE.status);
```

Comment:
`status` coming from api is of type `number`

Reply:
Fixed